lancero/cmd/acquire/.gitignore

```diff
@@ -0,0 +1 @@
+acquire
```

Comment:
Check this file. You will need to include the following lines in .gitignore:
```text
acquire
!acquire/
```

The first will ignore the binary called acquire AND all directories named acquire. Then the second says actually, don't ignore directories named acquire.

Maybe there is a more elegant way to do this, but I don't know it. (Don't build in the source directory??)

Reply:
Never mind. I misunderstood which .gitignore file this was.